chromium_src/components/sync/engine/cycle/status_controller.h

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_COMPONENTS_SYNC_ENGINE_CYCLE_STATUS_CONTROLLER_H_
+#define BRAVE_CHROMIUM_SRC_COMPONENTS_SYNC_ENGINE_CYCLE_STATUS_CONTROLLER_H_
+
+#define set_commit_result                                    \
+  set_last_server_error_message(const std::string& message); \
+                                                             \
+ public:                                                     \
+  std::string get_last_server_error_message() const;         \
```

Comment:
actually should return `const std::string&`

Reply:
yes, indeed, thanks, fixed in https://github.com/brave/brave-core/compare/cd423d681fd41cf3950850763706475699b7aab3..dafb93890912362658343c3063bad1f1d71bbc52